src/store/state/state-model.ts

```diff
@@ -137,7 +138,9 @@ export const stateModel: StateModelIF = {
         identifier: ''
       },
       nameRequest: {
-        legalType: ''
+        legalType: null,
+        nrNumber: '',
+        legalName: ''
```

Comment:
You might consider an "empty" nameRequest object that you can assign here.

Eg, https://github.com/bcgov/business-filings-ui/blob/091148a18ab9e62497eb2b12cbce256af609536f/src/interfaces/address-interfaces.ts#L26

Reply:
I like it! Implemented in next commit